.idems_app/deployments/plh/global.config.ts

```diff
@@ -14,7 +14,9 @@ const config: IDeploymentConfig = {
     sheets_folder_id: "1n221Zv9LYuwxmjhiboq8vhQg67_K9L5f",
     assets_folder_id: "1dp9QAQ84m8pm0IBQTSPXe1ramyynKPNn",
   },
-  app_data: {},
+  app_data: {
+    android_assets_subpath: "global/android"
```

Comment:
I would actually think of this the other way round and have a top level `android` config item that can be used to specify `icon_asset_path` and `splash_asset_path`

That shoule help keep code related to android-specific workflows a little more contained. In the future we could also expand to include other android-specific settings, like app id and name, and create additional workflows for configuring android deployment fully